cmd/fluxd/main.go

```diff
@@ -134,6 +133,8 @@ func main() {
 			logger.Log("err", err)
 			os.Exit(1)
 		}
+		clientset.Core()
```

Comment:
Looks like some dummy code I left in when exploring the k8s api! Oops!

_aside_: How on earth does this not error!

:+1: